bin/storybook/get-info.js

```diff
@@ -0,0 +1,74 @@
+// Figure out the Storybook version and view layer
```

Comment:
Looks like this file is missing the changes from https://github.com/chromaui/chromatic/pull/2070

Reply:
ok

Reply:
should be ok now